common/src/main/java/com/microsoft/identity/common/internal/util/AndroidKeyStoreUtil.java

```diff
@@ -87,20 +87,16 @@ public class AndroidKeyStoreUtil {
     private AndroidKeyStoreUtil() {
     }
 
-    private static KeyStore mKeyStore;
-
     private static final LongCounter sFailedAndroidKeyStoreUnwrapOperationCount = OTelUtility.createLongCounter(
             "failed_keystore_key_unwrap_operation_count",
             "Number of failed Android KeyStore unwrap operations"
     );
 
     private static synchronized KeyStore getKeyStore()
             throws KeyStoreException, CertificateException, NoSuchAlgorithmException, IOException {
-        if (mKeyStore == null){
-            mKeyStore = KeyStore.getInstance(ANDROID_KEY_STORE_TYPE);
-            mKeyStore.load(null);
-        }
-        return mKeyStore;
+        final KeyStore keyStore = KeyStore.getInstance(ANDROID_KEY_STORE_TYPE);
```

Comment:
Is this a lazy init? If not, i think we should make it so (i'm still thinking about OOM errors lol)

Reply:
No, it's new local object creation that's returned from the method. Just a util function